src/components/v2/AuthModal/WalletList/index.tsx

```diff
@@ -56,18 +56,19 @@ export const WalletList: React.FC<IWalletListProps> = ({ onLogin }) => {
 
       <div css={styles.footer}>
         <Typography variant="small2">
-          <Trans i18nKey="authModal.walletList.termsOfServiceLink">
-            By connecting a wallet, you agree to Venus&apos;{' '}
-            <a
-              href={VENUS_TERMS_OF_SERVICE_URL}
-              target="_blank"
-              rel="noreferrer"
-              css={styles.footerLink}
-            >
-              Terms of Service
-            </a>
-            .
-          </Trans>
+          <Trans
+            i18nKey="authModal.walletList.termsOfServiceLink"
+            components={{
+              anchor: (
```

Comment:
Since this is a component, how about capitalizing its name (same goes for other `Trans` instances)?

Reply:
works for me